requirements-dev.txt

```
@@ -1,16 +1,16 @@
 -r requirements.txt
-coverage==4.0.3
-flake8==2.4.1
-flake8-import-order==0.6.1
-funcsigs==0.4
-linecache2==1.0.0
-mccabe==0.3.1
-MiniMock==1.2.8
-mock==1.3.0
-nose==1.3.7
-pbr==1.8.1
-pep8==1.5.7
-pyflakes==0.8.1
-six==1.10.0
-traceback2==1.4.0
-unittest2==1.1.0
+coverage
+flake8
```

Flake can change. Nothing so frustrating as a failing CI because something got updated in PEP8 etc.

```suggestion
flake8<3.8
```